include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -225,6 +225,18 @@ class __scan_single_wg_dynamic_kernel;
 template <typename... Name>
 class __scan_copy_single_wg_kernel;
 
+template <typename _CustomName, typename _Index, typename _Range1, typename _Range2>
+__future<sycl::event>
+__parallel_copy_impl(sycl::queue& __q, _Index __count, _Range1&& __rng1, _Range2&& __rng2)
+{
+    auto __brick = oneapi::dpl::__internal::__pstl_assign{};
+    return oneapi::dpl::__par_backend_hetero::__parallel_for_impl<_CustomName>(
+        __q,
+        unseq_backend::walk_n_vectors_or_scalars<decltype(__brick)>{std::move(__brick),
+                                                                    static_cast<std::size_t>(__count)},
+        __count, std::forward<_Range1>(__rng1), std::forward<_Range2>(__rng2));
```

Comment:
```suggestion
    return oneapi::dpl::__par_backend_hetero::__parallel_for_impl<_CustomName>(
        __q,
        unseq_backend::walk_n_vectors_or_scalars<decltype(__brick)>{oneapi::dpl::__internal::__pstl_assign{},
                                                                    static_cast<std::size_t>(__count)},
        __count, std::forward<_Range1>(__rng1), std::forward<_Range2>(__rng2));
```

Looks like the `__brick` local variable and their `move` not needed...

Reply:
Sure, done with a minor tweak to fix the type provided to template args.